models/standard_vtol/standard_vtol.sdf.jinja

```diff
@@ -940,9 +943,11 @@
       <sdk_addr>INADDR_ANY</sdk_addr>
       <sdk_udp_port>14540</sdk_udp_port>
       <hil_mode>{{ hil_mode }}</hil_mode>
-      <hil_state_level>false</hil_state_level>
-      <enable_lockstep>true</enable_lockstep>
-      <use_tcp>true</use_tcp>
+      <hil_state_level>0</hil_state_level>
+      <send_vision_estimation>0</send_vision_estimation>
+      <send_odometry>1</send_odometry>
```

Comment:
Are these the options that get set in CMake?

Reply:
The {{ variable_name }} options are set when jinja script gets called, they either are explicitly set or use defaults.